src/katsdpcontroller/generator.py

```diff
@@ -536,6 +536,13 @@ def _make_fgpu(
                 default=stream.bandwidth,
                 initial_status=Sensor.Status.NOMINAL,
             ),
+            Sensor(
+                float,
+                f"{stream.name}.pass-bandwidth",
+                "Bandwidth over which the CBF produces a flat response" "Hz",
```

Comment:
Not sure what happened here. This needs to be two separate args
```
Sensor(
    ...,
    "Bandwidth over which the CBF [...]",
    "Hz",
    ...,
)
```

Reply:
Oops: I left out a comma, and Black then wrapped the two separate lines into one. I tend to believe that Python allowing string concatenation like this was a mistake, because it leads to bugs like this. Fixed now.

Reply:
>  I tend to believe that Python allowing string concatenation like this was a mistake, because it leads to bugs like this.

Right? I didn't know it could be done this why. Tried it out locally and it worked, but figured you wouldn't do something like that.